driver-core/src/main/com/mongodb/client/model/search/ConstructibleBsonFieldToManifoldAdapter.java

```diff
@@ -0,0 +1,48 @@
+/*
+ * Copyright 2008-present MongoDB, Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *   http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package com.mongodb.client.model.search;
+
+import com.mongodb.internal.client.model.AbstractConstructibleBsonField;
+import org.bson.BsonInt32;
+import org.bson.BsonString;
+import org.bson.conversions.Bson;
+
+import static org.bson.assertions.Assertions.notNull;
+
+final class ConstructibleBsonFieldToManifoldAdapter extends AbstractConstructibleBsonField<ConstructibleBsonFieldToManifoldAdapter>
```

Comment:
I think the name should relate to its usage - eg search.  

`SearchConstructibleBsonField` is clearer to me.

Reply:
Done.